src/AspNetCoreModuleV2/RequestHandler/outofprocess/forwardinghandler.cpp

```diff
@@ -18,9 +18,9 @@ PROTOCOL_CONFIG             FORWARDING_HANDLER::sm_ProtocolConfig;
 RESPONSE_HEADER_HASH *      FORWARDING_HANDLER::sm_pResponseHeaderHash = NULL;
 
 FORWARDING_HANDLER::FORWARDING_HANDLER(
-    _In_ IHttpContext                   *pW3Context,
-    _In_  HTTP_MODULE_ID                *pModuleId,
-    _In_ OUT_OF_PROCESS_APPLICATION     *pApplication
+    _In_ IHttpContext                  *pW3Context,
+    _In_ BOOL                           fOutOfProcessWebSocketSupported,
```

Comment:
the name can be as simple as fWebSocketSupported. No one cares about OutOfProcess here